### PR TITLE
fix wuxiaworld.com

### PR DESCRIPTION
**Pull request by alzamer2**
this update include:
* update for Bearer that in #1360, which will make the Bearer login work in both scraper and Browser method.
* username/email and password can be used to login now.
* the script will wait for "Time Until Free Chapter" msg until disappear and chapter will fully loaded.

PS-1: if in your account setting you enabled Auto-unlock, and script used browser method you will unlock all chapter you marked in range.
~PS-2:dont put locked chapter in scraping range if chapter auto-unlock is disable as that will hang/enter the script into infinite loop of waiting~
PS-2:its not recommended to put locked chapter in scraping range if chapter auto-unlock is disable as that will make script to wait longer to see if the chapter will unlocked or not (like 10-20sec for each locked chapter)

### CODE REVIEW

File: sources/en/w/wuxiacom.py [hunk 1]
```diff
@@ -19,17 +20,54 @@ class WuxiaComCrawler(BasicBrowserTemplate):
...
+            #time.sleep(10) # wait 10sec instead for waiting for elemnt in case login failed
+            #driver.find_element(By.CSS_SELECTOR, "h6 button").click()
+            #driver.find_element(By.CSS_SELECTOR, "input#Username").send_keys(email)
+            #driver.find_element(By.ID, "Password").send_keys(password)
+            #driver.find_element(By.NAME, "button").click()
+            #self.wuxia_cookies = self.get_cookies()
+            #self._restore_cookies()
+            #logger.debug("cookies: Email(%s)", str(self.cookies))
+            #logger.debug("headers: Email(%s)", str(self.headers))
+            try:
```
Comment (dipu-bd) [inline sources/en/w/wuxiacom.py:59]: please remove the commented out codes. this helps the codebase to stay clean.

Comment (alzamer2) [reply sources/en/w/wuxiacom.py:59]: sure
but what about headless? should i turn it on?

Comment (dipu-bd) [reply sources/en/w/wuxiacom.py:50]: no need. it will be automatically turned off for environments with no display

**Review by dipu-bd — APPROVED** [review]

**Comment by dipu-bd** [comment]
The changes looks good, but there are some minor issues with linting. Check here: https://github.com/dipu-bd/lightnovel-crawler/actions/runs/7156781251/job/19575165040?pr=2204

**Comment by alzamer2** [comment]
will look into it

**Pull request merged**